apps/paymaster-proxy/src/rpc/wrapPaymasterResponseIntoJsonRpcResponse.ts

```diff
@@ -0,0 +1,24 @@
+import { JsonRpcError } from '@/errors/JsonRpcError'
+import { PaymasterRpcError } from '@/errors/PaymasterError'
+import type { PaymasterResponse } from '@/paymaster/types'
+
+export const wrapPaymasterResponseIntoJsonRpcResponse = <T>(
+  response: PaymasterResponse<T>,
+  id: number,
+) => {
+  if (response.success === false) {
```

Comment:
we can probably turn all instances of `response.success === false` to `!response.success`